src/pages/apply.js

```diff
@@ -186,12 +217,14 @@ class ApplicationForm extends Component {
         const { uid, roles } = this.context;
         roles.applicant = true;
 
-        let firstName, email;
+        let firstName, email, newName;
         let responses = inputs.map(({ id: inputId, value }) => {
           const id = parseInt(inputId);
 
-          if (id === 1) firstName = value.split(" ")[0];
-          else if (id === 2) email = value;
+          if (id === 1) {
```

Comment:
This would be a good spot for comments on what the ID number means (had to read through the code for a bit to remember what it was lol). Tbh the best solution would be using a string as the ID but that would involve changing a lot of things (gg we have tech debt), would be good to track that as an issue (I guess we need a tech debt label now, congrats we're a real org)

Reply:
https://github.com/BUUPE/interview/issues/131